gap/examples.gd

```diff
@@ -48,3 +48,5 @@ DeclareOperation("PetersenGraph", [IsFunction]);
 DeclareConstructor("GeneralisedPetersenGraphCons", [IsDigraph, IsInt, IsInt]);
 DeclareOperation("GeneralisedPetersenGraph", [IsInt, IsInt]);
 DeclareOperation("GeneralisedPetersenGraph", [IsFunction, IsInt, IsInt]);
+
+DeclareGlobalFunction("DIGRAPHS_KingsGraph");
```

Comment:
Based on the conversation today, this should probably not be a global function, bur rather an operation (like Peterson above).